releasenotes/notes/opencensusagent.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: release-notes/v2
+kind: feature
+area: telemetry
+
+releaseNotes:
+  - |
+    **Updated** added support for OpenCensusAgent formatted trace export with configurable trace context headers.
```

Comment:
Should we add the new tracers that are now supported explicitly?

Reply:
Do you mean the contexts that are supported? This only adds a single tracer w/ multiple contexts propagation header options.

Reply:
I see. I think mentioning OpenCensus here and a doc for contexts make sense.